core/driver/sources/src/main/scala/org/virtuslab/ideprobe/config/IntelliJProvider.scala

```diff
@@ -0,0 +1,96 @@
+package org.virtuslab.ideprobe.config
```

Comment:
why not in ide.intellij?

Reply:
config package is for config classes that only exist for mapping purpose

Reply:
Done.